src/diffusers/pipelines/blip_diffusion/modeling_ctx_clip.py

```diff
@@ -0,0 +1,217 @@
+"""
+ Copyright (c) 2023, salesforce.com, inc.
+ All rights reserved.
+ SPDX-License-Identifier: BSD-3-Clause
+ For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+"""
```

Comment:
We have a specific format for authoring license headers. Let's follow that.

Reply:
Done

Reply:
It's not reflecting in the commits somehow.